packages/fx-core/src/question/other.ts

```diff
@@ -1066,3 +1066,48 @@ function oauthClientSecretQuestion(): TextInputQuestion {
     },
   };
 }
+
+export function syncManifestQuestionNode(): IQTreeNode {
+  return {
+    data: {
+      type: "group",
+    },
+    children: [
+      {
+        data: {
+          type: "folder",
+          name: QuestionNames.ProjectPath,
+          title: getLocalizedString("core.syncManifest.projectPath"),
+          cliDescription: "Project Path",
+          placeholder: "./",
+          default: "./",
+        },
+        condition: () => {
+          return true;
```

Comment:
Is the conditional required here? Looks like the question should always be shown, so may not need to set the condition?

Reply:
Yes, you are right. I will remove the condition here.